ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Ledger/Block.hs

```diff
@@ -130,6 +135,18 @@ instance (SimpleCrypto c, Serialise ext') => Serialise (SimpleBlock' c ext ext')
       body <- decode
       return (SimpleBlock hdr body)
 
+instance  ( Typeable c
+          , Typeable ext
+          , ConvertRawHash (SimpleBlock c ext)
+          , SerialiseNodeToClient (SimpleBlock c ext) (SimpleBlock c ext)
+          , SerialiseNodeToClient (SimpleBlock c ext) (Serialised (SimpleBlock c ext))
+          , SerialiseNodeToClient (SimpleBlock c ext) (GenTx (SimpleBlock c ext))
+          , SerialiseNodeToClient (SimpleBlock c ext) (ApplyTxErr (SimpleBlock c ext))
+          , SerialiseNodeToClient (SimpleBlock c ext) (SomeSecond BlockQuery (SimpleBlock c ext))
+          , SerialiseNodeToClient (SimpleBlock c ext) (LedgerConfig (SimpleBlock c ext))
+          , SerialiseResult       (SimpleBlock c ext) (BlockQuery (SimpleBlock c ext))
+          ) => SerialiseNodeToClientConstraints (SimpleBlock c ext)
```

Comment:
Did you need to generalize the `MockBlock` instance, or did it just seem like the right thing to do? I wouldn't be surprised if we only ever required `RunNode` etc for the `MockBlock` specialization of `SimpleBlock`, FWIW.

Reply:
Yeah, I didn't need to generalize. I was trying to avoid an orphan instance, but I've reverted this so as to keep the diff cleaner.